src/launcher/standalone_launcher/lib/README.md

```diff
@@ -0,0 +1,7 @@
+# lib
```

Comment:
Delete this file and `downloadInstaller.js`. We need to revisit where the installers live after this PR.